### DISCUSSION TITLE
Opera/Chrome vs Firefox versions

### DISCUSSION THREAD

**Discussion by markygnlg** [austencm/youtube-auto-like#87] [General]
HI! Love the extension, I've been using it for yonks for Opera.
I just installed it onto Firefox today but the version is 3.2.7, "Last Updated 02 July 2021" (today).
I found the link on your '[Code](https://github.com/austencm/youtube-auto-like)' page & it says "Firefox (by @Taknok)" - is that not you? Or is the Firefox version different from the Chrome version?
I ask because the options on Firefox look like the suggestions from "[Feature request: like at random part of the video #84](https://github.com/austencm/youtube-auto-like/issues/84)"
Thanks.

**Comment by markygnlg** [comment]
nvm, I just found the github for the Taknok version, which states:

vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv

Taknok / youtube-auto-like
forked from austencm/youtube-auto-like
...
Credits
    Austencm

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Thanks.

**Reply by austencm** [reply]
Yup! Separate extension, different maintainer. Forked years ago.
